.env.libretestnet

```diff
@@ -0,0 +1,91 @@
+# global
+STAGE=dev
+APP_NAME=eosio-dashboard
+
+# wallet
+WALLET_DATA=./wallet_data
+
+# postgres
+POSTGRES_USER=eoscr
+POSTGRES_PASSWORD=password
+POSTGRES_DB=localdb
+POSTGRES_DATA=./db_data
+
+# hasura
+HASURA_GRAPHQL_ENABLE_CONSOLE=true
+HASURA_GRAPHQL_DATABASE_URL=postgres://eoscr:password@postgres:5432/localdb
+HASURA_GRAPHQL_ADMIN_SECRET=myadminsecretkey
+HASURA_GRAPHQL_UNAUTHORIZED_ROLE=guest
+HASURA_GRAPHQL_ACTION_BASE_URL=http://hapi:9090
+
+# hapi
+HAPI_EOS_API_NETWORK_NAME=lilbre
+HAPI_EOS_API_ENDPOINT=https://libre-testnet.edenia.cloud
+HAPI_EOS_API_CHAIN_ID=b64646740308df2ee06c6b72f34c0f7fa066d940e831f752db2006fcc2b78dee
+HAPI_EOS_BASE_ACCOUNT=baseaccount
+HAPI_EOS_BASE_ACCOUNT_PASSWORD=PW...
+HAPI_EOS_MECHANICS_ACCOUNT=eosmechatero
+HAPI_EOS_MECHANICS_PASSWORD=PW...
+HAPI_EOS_WALLET_URL=http://localhost:8888
+HAPI_EOS_BP_JSON_ON_CHAIN=false
+HAPI_EOS_BP_JSON_ON_CHAIN_CONTRACT=producerjson
+HAPI_EOS_BP_JSON_ON_CHAIN_TABLE=producerjson
+HAPI_EOS_BP_JSON_ON_CHAIN_SCOPE=producerjson
+HAPI_HASURA_URL=http://hasura:8080/v1/graphql
+HAPI_HASURA_ADMIN_SECRET=myadminsecretkey
+HAPI_SERVER_PORT=9090
+HAPI_SERVER_ADDRESS=hapi
+HAPI_DATABASE_URL=postgres://eoscr:password@postgres:5432/localdb
+HAPI_SYNC_PRODUCERS_INTERVAL=86400
+HAPI_SYNC_PRODUCER_INFO_INTERVAL=1
+HAPI_SYNC_PRODUCER_CPU_INTERVAL=6
+HAPI_SYNC_EXCHANGE_RATE=86400
+HAPI_EOS_EXCHANGE_RATE_API=https://api.coingecko.com/api/v3/simple/price?ids=wax&vs_currencies=usd
+HAPI_COINGECKO_API_TOKEN_ID=BTC
+HAPI_REWARDS_TOKEN=LIBRE
+HAPI_SYNC_STATS_INTERVAL=60
+HAPI_SYNC_SCHEDULE_HISTORY_INTERVAL=86400
+HAPI_EOS_STATE_HISTORY_PLUGIN_ENDPOINT=
+HAPI_EOS_GET_FAUCET_ACCOUNT=
+HAPI_EOS_FAUCET_ACCOUNT=
+HAPI_EOS_FAUCET_ACCOUNT_PASSWORD=
+GOOGLE_APPLICATION_CREDENTIALS=
+HAPI_RE_CAPTCHA_PROJECT_ID=
+HAPI_PUBLIC_RE_CAPTCHA_KEY=
+HAPI_CREATE_ACCOUNT_ACTION_NAME=
+HAPI_EOS_MECHANICS_INCLUDE_TRANSACTION=
+
+#webapp
+PORT=3000
+REACT_APP_TITLE=Libre Testnet Network Dashboard
+REACT_APP_VERSION=dev
+REACT_APP_NAME=
+REACT_APP_DEFAULT_PRODUCER_LOGO=https://ipfs.eosio.cr/ipfs/QmcK3TPKZB7fRK5MYYcpUZuZeWpjWEhF1GUY64gGYrXU1K
+REACT_APP_FOOTER_LINKS=[{ "text": "Libre Website", "src": "https://libre.org/" },{ "text": "Libre Block Explorer", "src": "https://libre-testnet-explorer.edenia.cloud/" },{"text": "Documentation","src": "https://libre-chain.gitbook.io/"},{"text": "Libre Network Monitor","src": "https://libre-testnet.eosio.online"}]
+REACT_APP_EOS_RATE_LINK=
+REACT_APP_USE_REWARDS=true
+REACT_APP_USE_VOTES=true
+REACT_APP_USE_CPU_BENCHMARK=false
+REACT_APP_USE_BLOCK_PRODUCER_AGREEMENT_CONTRACT=true
+REACT_APP_HASURA_URL=http://localhost:8585/v1/graphql
+REACT_APP_EOS_API_NETWORK_NAME=libre-testnet
+REACT_APP_EOS_API_NETWORK_LABEL=Libre Testnet
+REACT_APP_EOS_API_NETWORK_LOGO=https://ipfs.eosio.cr/ipfs/QmcK3TPKZB7fRK5MYYcpUZuZeWpjWEhF1GUY64gGYrXU1K
```

Comment:
no longer using IPFS for this , use this URL instead

https://eosio.online/images/libre.png